### ISSUE TITLE
äüöß i not readable any more

### ISSUE THREAD

**Issue by sl5net** [maestrith/AHK-Studio#182]
I switched again too ahkstudio (i prefer) from Visual Studio Node. 
And all my German special Charakters (äüöß) are now not readable any more. 
I have used it in Shortcut and now the script has errors

**Comment by maestrith** [comment]
Can you upload the file? I believe that it is an encoding issue.

**Comment by sl5net** [comment]
its a spmal part of it.
http://doSqlWeb.de/demo.ahk
ahk file upload here is not allowed

**Comment by maestrith** [comment]
send it to maestrith@gmail.com

**Comment by sl5net** [comment]
yes prob a  encoding issue. notepad++ or visual studio node dont disturb the encoding. its ansi i think

**Comment by maestrith** [comment]
Yea, making AHK understand the difference between UTF characters in an Ansii file is not really something that I can handle (at least as far as I know) so if you change the encoding to UTF-8 it "Should" work fine.

**Comment by sl5net** [comment]
FIXED. i changed all files to utf8 ( and BTW deleted a line i my ahk .
i deleted: ParseWords := JEE_StrUtf8BytesToText( ParseWords )
no problems special characters in german now  👍
